src/components/AddTask/AddTask.jsx

```diff
@@ -34,8 +53,15 @@ export default function AddTask() {
             >
                 <TextField id="standard-basic" label="Add Task" />
                 <TextField id="standard-basic" label="Task Description" />
-                <BasicDatePicker />
-                <Button className="add-task-button" variant="contained" color="warning">Add Task</Button>
+                <BasicDatePicker onChange={dueDateInput}/>
+                <Button
+                className="add-task-button"
```

Comment:
Nitpick, these arg lines should be indented, otherwise, formatting is great, like the newlines.
Also great class naming, not sure if you have experience with CSS, but that dashed name scheme is widely used as a standard format: https://getbem.com/introduction/